tools/package_tools/mmdeploy_builder.py

```diff
@@ -173,6 +174,30 @@ def build_mmdeploy(cfg, mmdeploy_dir, dist_dir=None):
     _call_command(bdist_cmd, mmdeploy_dir)
 
 
+def build_mmdeploy_python(python_executable, cfg, mmdeploy_dir):
+    cmake_envs = cfg.get('cmake_envs', dict())
+
+    args = [f'-D{k}={v}' for k, v in cmake_envs.items()]
+    args.append(
+        f'-DMMDeploy_DIR={mmdeploy_dir}/build/install/lib/cmake/MMDeploy')
+
+    if sys.platform == 'win32':
+        build_cmd = 'cmake --build . --config Release -- /m'
```

Comment:
As first, we activate mmdeploy-3.6, so it always build the py36 whl on windows.

If we assume that there are mmdeploy-{3.6-3.9} envs, we may just activate one env before building the python package, and there is no need to check python_versions.